examples/plot_sampling_target_usage.py

```diff
@@ -0,0 +1,241 @@
+"""
+======================================================================
+Usage of the ``sampling_target`` parameter for the different algorithm
+======================================================================
+
+This example shows the different usage of the parameter ``sampling_target`` for
+the different family of samplers (i.e. over-sampling, under-sampling. or
+cleaning methods).
+
+"""
+
+# Authors: Guillaume Lemaitre <g.lemaitre58@gmail.com>
+# License: MIT
+
+from collections import Counter
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from sklearn.datasets import load_iris
+
+from imblearn.datasets import make_imbalance
+
+from imblearn.over_sampling import RandomOverSampler
+from imblearn.under_sampling import RandomUnderSampler
+from imblearn.under_sampling import TomekLinks
+
+print(__doc__)
+
+
+def plot_pie(y):
+    target_stats = Counter(y)
+    labels = list(target_stats.keys())
+    sizes = list(target_stats.values())
+    explode = tuple([0.1] * len(target_stats))
+
+    fig, ax = plt.subplots()
+    ax.pie(sizes, explode=explode, labels=labels, shadow=True,
+           autopct='%1.1f%%')
+    ax.axis('equal')
+
+
+###############################################################################
+# Creation of an imbalanced data set from a balanced data set
```

Comment:
I find it a bit strange that this examples starts with showing how to create imbalanced data, while I expected it to be about resolving an imbalance (and the example is already very long .. so not sure the added value)

Reply:
I had the same thought when I edited the previous example.
I think that we should move it to a specific example for the dataset and don't clutter the usage of the sampler